### PR TITLE
axp - Chapter 10 - Practical Reference Tricks

### PR DESCRIPTION
**Pull request by a-panayotopoulos**
Submitting early since I'll be on holiday next week

### CODE REVIEW

File: chapter10/axp/ex02 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+use Benchmark qw(timethese);
+
+chdir( $ARGV[0] // "~" );
+
+timethese( 5, { naive => \&naive, schwartzian => \&schwartzian } );
```
Comment (thatguy59) [inline chapter10/axp/ex02]: When I cheated and used your solutions as a sanity check (I attributed your work in a comment of mine so it's *totally* fine), it complained to me that 5 wasn't enough iterations for it to really report anything

Comment (a-panayotopoulos) [inline chapter10/axp/ex02]: Strange. Either we're working from slightly different library versions, or I ran it against a larger directory than you, and it took that into account. I get the same message if I reduce it much further, though.

Comment (a-panayotopoulos) [inline chapter10/axp/ex02]: Update: yeah, this might be based on runtime rather than number of iterations.

File: chapter10/axp/ex03 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+
+print foreach
+	map $_->[0],
+	sort { $a->[1] cmp $b->[1] }
+	map [ $_, lc s/[^\p{Ll}\p{Lu}]//gr ],
```
Comment (codavex) [inline chapter10/axp/ex03]: Nice non-use of the book's suggested transformation

Comment (a-panayotopoulos) [inline chapter10/axp/ex03]: heh, thanks

**Pull request merged**